include/picongpu/param/fieldSolver.param

```diff
@@ -43,6 +43,8 @@ namespace picongpu
          * Field Solver Selection (note <> for some solvers), all in namespace maxwellSolver:
          *  - Yee : Standard Yee solver approximating derivatives with respect to time and
          * space by second order finite differences.
+         *  - CCK<>: Cowan-Cole-Karkkainen Solver, Dispersion free solver in the direction of the smallest
```

Comment:
Correct spelling is 'Kärkkäinen'. But I don't know, whether umlauts are allowed by our coding guidelines.

Reply:
I wasn't sure about that either, so i left it out. But it can be added if its allowed

Reply:
ping @psychocoderHPC: umlauts, or suding 'ae', or just 'a'?

Reply:
ping @psychocoderHPC

Reply:
Only plain ASCII, no umlauts. So I guess we should use standard ä -> ae replacement.

Reply:
Ohh I just see, that openPMD standard used the ä -> a replacement. Well, than lets just follow their scheme in order to have consistent naming. Since you used that form already, I will mark this thread as resolved.